modyn/supervisor/internal/pipeline_executor/models.py

```diff
@@ -285,6 +286,7 @@ def df(self) -> pd.DataFrame:
                 "training_id",
                 "id_model",
                 "most_recent_model",
```

Comment:
also here

Reply:
I've replace all occurrences except from the ones in `/analytics` (those I will patch tomorrow)